examples/deployment/algob.config.js

```diff
@@ -27,10 +27,10 @@ let accounts = mkAccounts([
     // You should view your accounts and replace this with your own account.
     // Command to view account addresses (and their balances):
     // goal account list -d ~/.algorand-local/Node/
-    addr: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE",
```

Comment:
Please don't change the credentials. 
It's not needed. Have them in your local copy or create accounts that are the same in your network.
This is not a useful change.

Reply:
ok